bsi_zoo/tests/test_estimators.py

```diff
@@ -30,6 +31,7 @@
         (iterative_L1_typeII, 0.1, 1e-1, 5e-1, "full"),
         (iterative_L2_typeII, 0.1, 1e-1, 1e-1, "full"),
         (gamma_map, 0.2, 1e-1, 5e-1, "full"),
+        (lemur),
```

Comment:
Would also need to pass other parameters. Checks fail (see [this](https://github.com/braindatalab/BSI-Zoo/pull/29/checks)).